scripts/verify

```diff
@@ -124,9 +128,19 @@ function clean_and_exit() {
   [[ -z "$message" ]] || echo "$message"
 
   delete_namespace
+  if [[ ! -z "$deployer_role_name" ]]; then
+    kubectl delete clusterrolebinding "$deployer_role_name"
+  fi
   exit 1
 }
 
+# TODO(#344) Remove
```

Comment:
- Move this to the line where the flag is defined.
- Clarify: `TODO(#344) Remove this flag once we have better support`